core/crawler-conf.yaml

```diff
@@ -43,8 +43,6 @@ parsefilters.config.file: "parsefilters.json"
 urlfilters.config.file: "urlfilters.json"
 
 parser.htmlmapper.classname: "org.apache.tika.parser.html.IdentityHtmlMapper"
-parser.ignore.outlinks.outside.host: true
```

Comment:
Why the inconsistency if all other filters have their settings set in the JSON file?